styles/styles.css

```
@@ -426,23 +516,24 @@ h3 {
     background-repeat: no-repeat;
     background-size: contain;
     width: 24px;
-    height: 14px;
 }
 
 .social-link-button-tg {
     background-image: url(../images/telegram.png);
     background-repeat: no-repeat;
     background-size: contain;
     width: 18px;
-    height: 15px;
 }
 
 .social-link-button-youtube {
     background-image: url(../images/youtube.png);
     background-repeat: no-repeat;
     background-size: contain;
     width: 22px;
-    height: 17px;
+}
+.social-link {
+    display: flex;
+    justify-content: flex-start;
 }
```

```suggestion
.social-link {
    display: flex;
}
```
значение по умолчанию - можно не писать